src/patternfly/components/Masthead/masthead.scss

```diff
@@ -0,0 +1,234 @@
+  --pf-c-masthead--m-display-inline__main--PaddingTop: 0;
+  --pf-c-masthead--m-display-inline__main--PaddingBottom: 0;
+  --pf-c-masthead--m-display-inline__main--MarginRight: calc(var(--pf-global--gutter--md) / 2); // split gutter
+  --pf-c-masthead--m-display-inline__main--before--BorderBottom: 0;
+  --pf-c-masthead--m-display-inline__content--Order: 0;
+  --pf-c-masthead--m-display-inline__content--PaddingTop: 0;
+  --pf-c-masthead--m-display-inline__content--PaddingBottom: 0;
+  --pf-c-masthead--m-display-inline__content--MarginLeft: calc(var(--pf-global--gutter--md) / 2); // split gutter
+  --pf-c-masthead--m-display-inline__content--c-nav--m-horizontal--MarginRight: 0;
+  --pf-c-masthead--m-display-inline__content--c-nav--m-horizontal--MarginLeft: 0;
+
+  // main border
+  --pf-c-masthead__main--before--Right: calc(var(--pf-c-masthead--inset) * -1);
+  --pf-c-masthead__main--before--Left: calc(var(--pf-c-masthead--inset) * -1);
+  --pf-c-masthead__main--before--BorderBottomWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-masthead__main--before--BorderBottomColor: var(--pf-global--palette--black-600);
+
+  // toggle
+  --pf-c-masthead__toggle--MarginRight: calc(var(--pf-global--gutter--md) / 2);
+
+  // light 100
+  --pf-c-masthead--m-light-100--BackgroundColor: var(--pf-global--BackgroundColor--100);
+  --pf-c-masthead--m-light-100__main--BorderBottomColor: var(--pf-global--palette--black-200);
+
+  // light 200
+  --pf-c-masthead--m-light-200--BackgroundColor: var(--pf-global--BackgroundColor--200);
+  --pf-c-masthead--m-light-200__main--BorderBottomColor: var(--pf-global--palette--black-300);
+
+  // set layout to stack by default
+  @include pf-c-masthead--m-display-stack;
+
+  // set layout to inline at medium breakpoint
+  @media screen and (min-width: $pf-global--breakpoint--md) {
+    @include pf-c-masthead--m-display-inline;
+  }
+
+  @media screen and (min-width: $pf-global--breakpoint--xl) {
+    --pf-c-toolbar--m-page-insets--inset: var(--pf-c-toolbar--m-page-insets--xl--inset);
+  }
+
+  @include pf-t-dark; // force the container to follow the dark theme
+
+  position: relative;
+  display: flex;
+  flex-wrap: var(--pf-c-masthead--FlexWrap);
+  align-items: center;
+  min-width: 0;
+  padding-right: var(--pf-c-masthead--PaddingRight);
+  padding-left: var(--pf-c-masthead--PaddingLeft);
+  background-color: var(--pf-c-masthead--BackgroundColor);
+
+  &.pf-m-light-100 {
+    @include pf-t-light;
+
+    --pf-c-masthead--BackgroundColor: var(--pf-c-masthead--m-light-100--BackgroundColor);
+    --pf-c-masthead__main--BorderBottomColor: var(--pf-c-masthead--m-light-100__main--BorderBottomColor);
+  }
+
+  &.pf-m-light-200 {
+    @include pf-t-light;
+
+    --pf-c-masthead--BackgroundColor: var(--pf-c-masthead--m-light-200--BackgroundColor);
+    --pf-c-masthead__main--BorderBottomColor: var(--pf-c-masthead--m-light-200__main--BorderBottomColor);
+  }
+
+  &.pf-m-page-insets {
```

Comment:
should this be the default, assuming the default use case of the masthead is in the page?